### ISSUE TITLE
Is there an option to suppress citations on first usage of \acs?

### ISSUE THREAD

**Issue by obreitwi** [cgnieder/acro#185]
Currently, if I have an acronym defined with a `cite=` entry and I expand it using `\acs`, the citation will be appended.

This looks a bit weird for nested acronyms in which the long form depends on the short form of another acronym.

Is there any way to prevent appending the citation other than `\acuse`-ing the citation beforehand and resetting it afterwards?

Thank you!

**Comment by cgnieder** [comment]
Can you make an example illustrating your problem? Then I might be able to give a proper answer.

**Comment by obreitwi** [comment]
While making an example (and thereby re-reading the manual _yet_ again), I discovered my error: I was using the unstarred version of `\acs` which rightfully inserts the citation; switching to `\acs*` when using nested citations fixed the error.

Not that it matters now, but I uploaded what I was trying to do [here](https://gist.github.com/obreitwi/550ddb4de235cb7d479c55e7292bcad6) which produces [this result](https://breitwieser.eu/tmpodrom/2Io-IkngCeUK-TfI/issue.pdf).

I am very sorry for the noise, thank you for `acro`!
